go/vm/lfvm/interpreter_test.go

```diff
@@ -69,24 +69,39 @@ func getContext(code Code, data []byte, runContext vm.RunContext, stackPtr int,
 
 // Test UseGas function and correct status after running out of gas
 func TestGasFunc(t *testing.T) {
-	ctx := getEmptyContext()
-	ctx.gas = 100
-	ok := ctx.UseGas(10)
-	if !ok {
-		t.Errorf("expected not failed useGas function, got failed")
-	}
-	if ctx.gas != 90 {
-		t.Errorf("expected gas in context is 90, got %d", ctx.gas)
-	}
-	ok = ctx.UseGas(100)
-	if ok {
-		t.Errorf("expected failed useGas function, got ok")
-	}
-	if ctx.gas != 90 {
-		t.Errorf("expected gas in context is 90 also after failing, got %d", ctx.gas)
+
+	tests := []struct {
+		name      string
+		gas       vm.Gas
+		amount    vm.Gas
+		wantedGas vm.Gas
+		expected  bool
+		status    Status
```

Comment:
Renaming those fields would help readability:
- `gas` => `initialGas`
- `amount` => `cost` or `delta`
- `wantedGas` => `resultingGas`
- `expected` => not needed at all; can be computed as `initialGas` >= `amount` 
- `status` => not need, can be derived from other parameters; in particular `!expected` => `OUT_OF_GAS`